plugin/harpoon.vim

```diff
@@ -12,7 +12,7 @@ fun! GotoBuffer(ctrlId)
     let contents = g:win_ctrl_buf_list[a:ctrlId]
     if type(l:contents) != v:t_list
         " Create the terminal
-        terminal
+        exe has("nvim") ? "terminal" : "terminal ++curwin"
```

Comment:
what is `++curwin`?

Reply:
Normally if you do a `:terminal` in vim it will create the terminal
in a split. So you can give it the `++curwin` flag to create the
terminal in the current window. So it works likes the neovim terminal.